hphp/test/slow/include/fast-stat-cache.php.expect

```diff
@@ -0,0 +1 @@
+OldOldNew
```

Comment:
This is a good example.  faststatcache changed the semantics of the language, right?  Without the faststatcache, I expect output 'OldNewNew' instead.  In a sense, this makes the implementation 'incorrect'.  Do you agree?

If that is the case, I think we need to document clearly that the option makes unsafe assumptions and should only be enabled when people know what they are doing.  Maybe rename the option to 'UnsafeStatCache' instead.